src/components/SearchModal/ManageLists.tsx

```diff
@@ -160,8 +169,19 @@ const ListRow = memo(function ListRow({ listUrl }: { listUrl: string }) {
 
   if (!list) return null
 
+  const { label } = CHAIN_INFO[chainId ? chainId : SupportedL1ChainId.MAINNET]
+
+  const hoverText = t`This token list currently has ${activeTokensOnThisChain} ${label} tokens in it`
```

Comment:
You could use a plural form here. From the [docs](https://lingui.js.org/#quick-overview):

```
<Plural  value={activeTokensOnThisChain}
        one={
            This token list currently has one ${label} tokens in it
        }
        other={`This token list currently has ${activeTokensOnThisChain} ${label} tokens in it`}
```

Reply:
v cool

Reply:
Handled plurals, but didn't end up using the Plural component because I need a string for the title property.